package.json

```diff
@@ -30,28 +37,45 @@
     "decode-uri-component": "^0.2.0",
     "eslint-plugin-jsdoc": "^15.6.1",
     "express": "^4.16.3",
-    "express-basic-auth": "1.1.7",
+    "express-basic-auth": "^1.1.7",
     "express-openapi": "^4.6.5",
     "express-winston": "^3.2.1",
+    "forever": "^1.0.0",
     "js-yaml": "^3.13.1",
     "jsonapi-serializer": "^3.5.6",
     "lodash": "^4.17.13",
     "moment": "^2.22.2",
     "query-string": "^6.2.0",
     "simple-git": "^1.96.0",
+    "source-map-support": "^0.5.12",
     "swagger-parser": "^6.0.3",
     "winston": "^3.2.1",
     "winston-daily-rotate-file": "^3.10.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.4.3",
+    "@babel/plugin-proposal-optional-chaining": "^7.2.0",
+    "@babel/plugin-transform-runtime": "^7.4.3",
+    "@babel/preset-env": "^7.4.3",
+    "@babel/preset-flow": "^7.0.0",
+    "@babel/runtime": "^7.4.3",
```

Comment:
I think this plugin actually needs to be in our dependencies instead of devDependencies